pulp_rpm/app/kickstart/treeinfo.py

```diff
@@ -182,15 +166,88 @@ def create_treeinfo(distribution_tree):
     return f
 
 
-class KickstartData:
+class PulpTreeInfo(TreeInfo):
```

Comment:
The solution was to extend `productmd.TreeInfo` class and handle errors by sections.
At first, I tried just to use `SortedConfigParser` and due that I was not able to download some images.
Then I realized `productmd.TreeInfo` has some strategies to parse the data, and I was missing it, and the problem we were having before was that `productmd.TreeInfo`  tries to validate all sections: https://github.com/release-engineering/productmd/blob/f6d73c9822a6451ef4e3af98393bf3935102667a/productmd/treeinfo.py#L104-L133
So I'm changing this behavior to just validate the sections that are present